packages/core/test/session.test.ts

```diff
@@ -87,6 +88,22 @@ describe('session', () => {
     expect(result._watermarks).toEqual({ high: Number.MAX_VALUE, low: Number.MAX_VALUE })
   })
 
+  it('run should set expected apiTelemetryConfig with api equals to TELEMETRY_APIS.AUTO_COMMIT_TRANSACTION', async () => {
```

Comment:
```suggestion
  it('run should set protocol option apiTelemetryConfig.api to TELEMETRY_APIS.AUTO_COMMIT_TRANSACTION', async () => {
```
or something like it. I don't quite understand the sentence that was there.